src/screens/WalletConnect/WalletConnectList.js

```diff
@@ -67,13 +68,13 @@ const style = StyleSheet.create({
   },
   sessionName: {
     marginBottom: 8,
-    color: 'rgba(0, 0, 0, 0.87)',
+    color: COLORS.textColorShadowDark,
     fontSize: 12,
     fontWeight: 'bold',
   },
   sessionData: {
     marginBottom: 8,
-    color: 'rgba(0, 0, 0, 0.57)',
+    color: COLORS.textColorShadow,
```

Comment:
Changed from `0.57` opacity to `0.5` opacity

Reply:
Do we have a constant with `0.6`?

Reply:
Changed to `0.6` opacity on [a6b16fd](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/a6b16fdf489a4652038ac614da23fe5485e62020)